test/client.js

```diff
@@ -791,6 +791,60 @@ describe('Client', () => {
             client.disconnect();
             await server.stop();
         });
+
+        describe('empty response handling', () => {
+
+            [
+                {
+                    testName: 'handles empty string, no content-type',
+                    handler: () => '',
+                    expectedPayload: ''
```

Comment:
The more I think about this, the more I'm leaning towards handling `''` and `null` in the same way, when `content-type` is not defined... I think `nes` _server_ should return `null` for both.

Reply:
Then again... Maybe if the response is an empty _string_, Hapi should explicitly set `text/html` (or `text/plain`?) content-type?

Reply:
https://github.com/hapijs/hapi/issues/3782